frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -102,7 +104,7 @@ const DisplayList = ({
   if (displayShoutouts === undefined) {
     return <Loader active size="big" content="Loading..." />;
   }
-  if (displayShoutouts.length === 0)
+  if (displayShoutouts?.length === 0)
```

Comment:
I think we don't need this bc we check if `displayShoutouts` is `undefined` before.